examples/kotlinExample/build.gradle

```diff
@@ -45,11 +45,18 @@ android {
     }
 }
 
+repositories {
+    // for ReLinker
+    maven { url 'https://jitpack.io/'}
+}
+
 dependencies {
     compile "org.jetbrains.kotlin:kotlin-stdlib:${kotlin_version}"
     compile "org.jetbrains.kotlin:kotlin-reflect:${kotlin_version}"
-    compile 'org.jetbrains.anko:anko-sdk15:0.7.2'
-    compile "io.realm:realm-android-library:${version}@aar"
+    compile 'org.jetbrains.anko:anko-sdk15:0.8.2'
+    compile ("io.realm:realm-android-library:${version}@aar") {
+        transitive=true
+    }
     compile "io.realm:realm-annotations:${version}"
```

Comment:
We can't remove this since we specify `aar` dependency instead of default `pom` dependency.

If remove this, we got `java.lang.NoClassDefFoundError: Failed resolution of: Lcom/getkeepsafe/relinker/ReLinker;`